lib/travis/build/script/hack.rb

```diff
@@ -0,0 +1,68 @@
+module Travis
+  module Build
+    class Script
+      class Hack < Php
+        DEFAULTS = {
+          hhvm: 'hhvm', # lts
+          php: '7.2',
+        }
+
+        VALID_HHVM = %w(
+          hhvm
+          hhvm-dbb
+          hhvm-nightly
+          hhvm-nightly-dbg
+          hhvm-dev-nightly
```

Comment:
This doesn't exist

Reply:
Then it should be removed from the documentation (see above).